features/collection/snmp-collector/src/main/java/org/opennms/netmgt/collectd/AliasedResource.java

```diff
@@ -96,13 +96,12 @@ public String getDomain() {
             return Integer.toString(getIfInfo().getNodeId());
         } else if ("nodelabel".equalsIgnoreCase(m_domain)) {
             try {
-                return new NodeLabelJDBCImpl().retrieveLabel(getIfInfo().getNodeId()).getLabel();
-            } 
-            catch (Throwable e) {
+                return new NodeLabelDaoImpl().retrieveLabel(getIfInfo().getNodeId()).getLabel();
```

Comment:
Is there a reason we switch from the `NodeLabelJDBCImpl` to the `NodeLabelDaoImpl` here?

Reply:
I wanted to avoid updating `opennms-webapp` which depends on `NodeLabelJDBCImpl` (in `opennms-services`) - I would have had to pull the `jdbcimpl ` up into thresholding to prevent a circular dependency.
NodeLabelJDBCImpl has also been deprecated, so switching to the hibernate impl seemed to be both easy, and sensible.

Reply:
reverted this change but moved NodeLabelJDBCImpl into this package.